Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "tree-sitter-grep"
+name = "tree_sitter_lint_tree-sitter-grep"
```

Comment:
Per helixbass/tree-sitter-lint#4, updating this to allow publishing that crate while this hasn't "landed"